test/solve.jl

```diff
@@ -1,11 +1,27 @@
 @testset "Solving linear system" begin
 
-    @testset "Problem size: $n x $n. Matrix type: $m. Element type: $elty" for n in (1,2,3,4),
+    # I see an inference error in the combined testset below. I can't reproduce
+    # at the REPL or in a function for individual n, etc... speculatively, it
+    # might be reusing A, b with different types in the same (non-toplevel)
+    # scope, for which I've come accross inference bugs in the past.
```

Comment:
I think it's not a problem in practice, or at least I couldn't find any evidence of a problem.